package.json

```diff
@@ -12,7 +12,8 @@
   "scripts": {
     "assert": "exit 0",
     "build": "webpack --config ./client/webpack.config.js",
-    "dev": "nodemon --watch server & yarn build --watch",
+    "webpack:watch": "webpack --watch --config ./client/webpack.config.js",
+    "dev": "nodemon --watch server & yarn webpack:watch",
```

Comment:
I've played around with these settings and the `--watch` should work the way it was. Let's undo this change and explore further if it isn't working the way we expect.